apps/theming-designer/src/components/SamplesCardHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Text, ITextProps } from '../../../../packages/office-ui-fabric-react/lib/Text';
+import './SamplesCardHeader.scss';
+
+interface ISamplesCardHeaderProps {
```

Comment:
Nit: A more intuitive API would probably be to accept `children` instead of `label`. That way, a caller can do this instead:

    <SampleCardHeader>Input</SampleCardHeader>